docs/builtins/Time/format-time.md

```diff
@@ -3,6 +3,21 @@
 Use `format-time` to format a `time` value using a specified `format`.
 The `format-time` function is useful for converting time values to human-readable formats in Pact contracts.
 
+The following table provides a summary of common time formatting codes:
+
+| Format | Purpose |
+| --- | --- |
+| %Y | Year with no padding. |
+| %m | Month of the year, zero-padded to two characters, "01"–"12" |
+| %d | Day of the month, zero-padded to two characters, "01"–"31" |
+| %H | Hour of the day using a 24-hour clock, zero-padded to two characters, "00"–"23" |
+| %M | Minute of of the hour, zero-padded to two characters, "00"–"59" |
+| %S | Second of the minute, zero-padded to two characters, "00"–"60" |
+
+There are many other formatting options.
+For example, you can replace the numeric representing the month of the year with the short or long name for the month.
+For information about all of the formats supported, see [Time formats](/pact-5/Time/time-functions#time-formats).
```

Comment:
Is this link still required?

Reply:
This link is to the topic in the new site, so I'll hide this for now and add the better organized formatting table from the new site to the current docs for both the `format-time` and `parse-time` functions.